defi/src/protocols/data2.ts

```diff
@@ -26716,11 +26716,11 @@ const data2: Protocol[] = [
       "tBTC is Threshold's decentralized and permissionless bridge to bring BTC to the Ethereum network. tBTC has been designed to allow bitcoin holders to participate in Ethereum's Decentralized Finance (DeFi) applications. Users wishing to utilize their bitcoin on Ethereum can use the tBTC decentralized bridge to deposit their bitcoin into the system and get a minted tBTC token in their Ethereum wallet",
     chain: "Ethereum",
     logo: `${baseIconsUrl}/tbtc.png`,
-    audits: "2",
+    audits: "4",
```

Comment:
2 just means that its audited, its not the number of audits

should leave as is